tsconfig.json

```diff
@@ -17,5 +17,8 @@
     "lib": ["es2019"],
     "resolveJsonModule": true
   },
-  "include": ["src/**/*"]
+  "include": [
+    "src/**/*",
+    "proxy.d.ts"
```

Comment:
I don't know where's the best place to put this type file, probably not at the root, waiting for some suggestion?

Reply:
You can put it in `src/` or if you prefer to have all declarations in a folder you may create a `types` folder